spec/book_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+# Test for initialize method
+describe Book do
+  it 'should create instance of the book class with 2 parameters' do
+    book = Book.new('Title', 'Author')
+    expect(book).to be_an_instance_of Book
+  end
+end
+
+# Test for title
+describe '#title' do
+  it 'should return the right title' do
+    book = Book.new('My Title', 'My Author')
+    expect(book.title).to eq('My Title')
+  end
+end
+
+# Test for author
+describe '#author' do
+  it 'should return the right author' do
+    book = Book.new('My Title', 'My Author')
+    expect(book.author).to eq('My Author')
+  end
```

Comment:
- [ ] @team great work on your tests, kindly add a test for your add_rental method, its a project requirement to add tests for all your methods
![image](https://user-images.githubusercontent.com/61408860/218126495-33827df2-a6eb-4282-802e-549faa5df3a0.png)

Reply:
- Well resolved 👍🏽